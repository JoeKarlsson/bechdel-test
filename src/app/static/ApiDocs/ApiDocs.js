import React from 'react';

const ApiDocs = () => (
	<div className="container">
		<h1>api documentation</h1>
	</div>
);

export default ApiDocs;
