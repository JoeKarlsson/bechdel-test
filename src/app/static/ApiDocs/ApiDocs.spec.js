import React from 'react';
import ApiDocs from './ApiDocs';

describe('Api Docs Page', () => {
	describe('rendering', () => {
		it('match the snapshot', () => {
			const wrapper = shallow(<ApiDocs />);
			expect(wrapper).toMatchSnapshot();
		});
	});
});
