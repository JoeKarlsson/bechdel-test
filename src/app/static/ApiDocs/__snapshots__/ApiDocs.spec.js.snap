// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Api Docs Page rendering match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
}
`;
